README.md

```diff
@@ -24,6 +24,7 @@ sonata_admin_search:
     admin_finder_services:
         my_admin.id: # Admin service id
             finder: Id for a FOS finder service that should be used # Finder service
+            actions : [list] #[Optional] actions where elastic search has to be enabled
```

Comment:
can you please put this comment at the ``Configuration.php``, too.

use ``->info('my funky help')``